### DISCUSSION TITLE
Why Django version required has changed to be <3.0?

### DISCUSSION THREAD

**Discussion by marcosalcazar** [cyface/django-termsandconditions#185] [General]
Hello! I've noticed with v2.0.7 the Django version supported has changed from "^=2.2" to 'Django>=2.2,<3.0'. We've been using this package with Django 3.0 just fine. What's the reason behind this decision? Thanks!

**Comment by cyface** [comment]
Hello! Was not intentional. 👍🏻   May be an artifact of switching to Poetry for the release manager. I'll take a look.

**Reply by marcosalcazar** [reply]
Thanks!

**Reply by cyface** [reply]
OK, 2.0.8 is out that should set this back.  Sorry about that - one character off in the .toml file. 👍🏻

**Reply by marcosalcazar** [reply]
Thank you so much! And for tackling this so quickly!
